docs/content/contributing/maintainers-guidelines.md

```diff
@@ -7,128 +7,121 @@ description: "Interested in contributing more to the community and becoming a Tr
 
 ![Maintainer's Guidelines](../assets/img/maintainers-guidelines.png)
 
-Note: the document is a work in progress.
-
 Welcome to the Traefik Community.
-This document describes how to be part of the core team
-together with various responsibilities
-and guidelines for Traefik maintainers.
+
 We are strongly promoting a philosophy of openness and sharing,
 and firmly standing against the elitist closed approach.
 Being part of the core team should be accessible to anyone motivated
 and wants to be part of that journey!
 
-## Onboarding Process
+## Becoming a Maintainer
+
+Before a contributor becomes a maintainer, they should meet the following requirements:
```

Comment:
at least for me it is not clear who is really maintainer for Traefik currently. I have been just looking who has "traefik" organisation in their Github badges. However, it would be worth of thinking should there be somekind of place where this information is available? If I take example from `kubernetes` they use [OWNERS](https://github.com/kubernetes/kops/blob/master/OWNERS) files. Github do support also [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).

Why I am asking this? I am seeing that like PRs are waiting for weeks / months to get any progress. It is not sign of healthy OpenSource project. As an example: I am external contributor for Traefik, but I would not like to ask reviews all the time. That should happen somehow automatically in some period. Over n weeks/months old PRs should be either closed (with comments) or make/ask the needed changes.

Reply:
> at least for me it is not clear who is really maintainer for Traefik currently. 

Maintainers are listed in the [maintainers](https://github.com/traefik/traefik/blob/master/docs/content/contributing/maintainers.md) section in the documentation (which the PR updates and restricts to active maintainers). 

Yes, we probably should make it a bit more obvious, including it in the [README](https://github.com/traefik/traefik#readme) for example.

> It would be nice that PRs are automatically assigned to someone where the (external) contributor can ask for help and help for reviewing. Now all of this does not happen automatically. So I would recommend using CODEOWNERS (it does that).

About the CODEOWNERS, it's a nice idea we should discuss as well, it might help streamlining the work for some requests (even though there's no specific maintainer in charge of a given part of traefik today).

> Why I am asking this? I am seeing that like PRs are waiting for weeks / months to get any progress. It is not sign of healthy OpenSource project.

I (speaking for myself) agree this is not ideal, even though there are many underlying reasons (incl. bandwidth, we are for now not so many maintainers) that some PRs are kept open with not much progress. This is one of the points this guideline update tries to address, at the very least in the philosophy.

It's only a small tweak in the text, but the philosophy is to document activity on PRs. There are many times in the past where PRs where discussed internally with maintainers, without proper documentation on the given PR (we could take decisions, for example, to "wait" until we saw more activity. But with no documentation of that decision, it's unclear).

Reply:
> ...  maintainer.md in documentation. I did not know that. Sorry for that. Does that file contains now everyone that can give like PR reviews?

Yes, maintainers there are everyone who can review PRs.
The process we follow today for PRs is described [here from the perspective of contributors](https://github.com/traefik/contributors-guide/blob/master/pr_guidelines.md) and [there from the perspective of the process](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md)

Reply:
FYI, I created a [PR to improve the README](https://github.com/traefik/traefik/pull/10230) in that regard